src/Context/Node/NodeContext.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @author Sergii Bondarenko, <sb@firstvector.org>
+ */
+namespace Drupal\TqExtension\Context\Node;
+
+use Drupal\TqExtension\Context\RawTqContext;
+
+class NodeContext extends RawTqContext
+{
+    /**
+     * @When /^I (visit|view|edit) "([^"]+)" node of type "([^"]+)"$/
+     */
+    public function iVisitNodePageOfType($operation, $title, $type) {
+        if ($operation == 'visit') {
+            $operation = 'view';
+        }
+        $query = new \EntityFieldQuery();
+        $result = $query
+            ->entityCondition('entity_type', 'node')
+            // @todo: Add support for CT label.
+            ->entityCondition('bundle', strtolower($type))
+            ->propertyCondition('title', $title)
+            ->range(0, 1)
+            ->execute();
+
+        if (empty($result['node'])) {
+            $params = array(
```

Comment:
`$params` array is unnecessary. In this case, it should be used as value for `format_string` but we even don't need this Drupal functionality here. Just `sprintf()`.